imagespace/server/imagesearch_rest.py

```diff
@@ -101,16 +102,23 @@ def filenameUpper(filename):
             query += ' AND (%s)' % ' OR '.join(['%s:[.7 TO *]' % key
                                                 for key in classifications])
 
+        qparams = {
+            'wt': 'json',
+            'hl': 'true',
+            'hl.fl': '*',
+            'q': query,
+            'start': offset,
+            'rows': limit
+        }
+
+        # Give plugins a chance to adjust the Solr query parameters
+        event = events.trigger('imagespace.imagesearch.qparams', qparams)
```

Comment:
I pushed a commit that slightly changed this. I want plugins to be able to completely override `qparams` - this way the logic for how a certain field has to be changed can be handled within a plugin.

For example if I wanted to append something to a certain field, and overwrite another - that would be impossible with the former method.

The downside is we need to be careful because plugins could conflict (though this could have happened anyway).